workers/unity/Assets/Gdk/Legacy/Editor/BuildSystem/Util/SpatialCommandRunner.cs

```diff
@@ -27,5 +28,24 @@ internal static void RunSpatialCommand(string args, string description)
                     errOut, description));
             }
         }
+
+        private static string GetOutput(TextReader reader)
+        {
+            var line = "";
+            var output = "";
+            while (line != null)
+            {
+                try
+                {
+                    line = reader.ReadLine();
+                    output += line;
+                }
+                catch
+                {
+                    line = null;
```

Comment:
Just `break` out of the while loop?

Reply:
changed the whole code after speaking to @jared-improbable